python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -66,6 +66,16 @@ def copy_scaffold(
         dbt_project_yaml: Dict[str, Any] = yaml.safe_load(fd)
         dbt_project_name: str = dbt_project_yaml["name"]
 
+    dbt_project_dir_relative_path = Path(
+        os.path.relpath(
```

Comment:
Using `os.path.relpath` in place of `Path.relative_to` since the former is less restrictive in calculating relative paths. See more https://docs.python.org/3/library/pathlib.html#id4.